sdk/eventgrid/Azure.Messaging.EventGrid/tests/SystemEventTests.cs

```diff
@@ -39,10 +40,9 @@ public void MappingContainsAllSystemEvents()
                 }
 
                 ValidateName(systemEvent.Name);
-                Assert.IsTrue(
-                    SystemEventExtensions.s_systemEventDeserializers.Values.Any(
-                        f => f.Method.ReturnType == systemEvent), systemEvent.Name);
+                Assert.IsNotNull(SystemEventExtensions.AsSystemEventData(systemEvent.Name, JsonDocument.Parse("{}").RootElement));
```

Comment:
Honestly, I think the previous test did never reach `Assert.IsTrue` because the above `GetConstructors` makes all events skipped. At least this is what happened on my machine when I debugged the main branch version of this test because I was thinking my added Assert should not pass